gb-cpu/src/memory/area.rs

```diff
@@ -0,0 +1,35 @@
+pub mod rom;
+pub mod wram;
+
+use super::consts;
+pub use wram::Wram;
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum Area {
```

Comment:
Maybe rework `Area` to give for each item a `u16` and remove the `address` from relative ?

Reply:
Yeah, i think that may be better, i'ill look into it, great catch!